proxy/proxy.go

```diff
@@ -671,6 +699,22 @@ func (p *Proxy) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	ctx := newContext(w, r, p.flags.PreserveOriginal(), p.metrics)
 	ctx.startServe = time.Now()
 
+	if p.openTracing {
+		carrier := ot.HTTPHeadersCarrier(r.Header)
+		tracer := ot.GlobalTracer()
+		wireContext, err := tracer.Extract(ot.HTTPHeaders, carrier)
+		var span ot.Span
+		if err == nil {
+			span = tracer.StartSpan("ingress", ext.RPCServerOption(wireContext))
```

Comment:
what does 'ingress' mean here? Is it a reference to the Kubernetes Ingress? If yes, then I think this should be a bit more general, or if needed should come from the settings, because ingress is not the only use case of skipper.

Reply:
Yes, that's were it came from.